src/auth.py

```diff
@@ -107,6 +114,16 @@ def rq_get(request: Request, key: str) -> str:
         return j.get(key, "")
     return ""
 
+def get_facebook_public_key():
+    # Get the public key from Facebook's JWKS endpoint
+    response = requests.get(FACEBOOK_JWT_ENDPOINT)
```

Comment:
We need some kind of error handling around this, i.e. a try-catch block, and a way to return an error status if the fetch was not successful.